src/scenarios/netandroid/pre.py

```diff
@@ -0,0 +1,40 @@
+'''
+pre-command
+'''
+import shutil
+import sys
+from performance.logger import setup_loggers, getLogger
+from shared import const
+from shared.mauisharedpython import remove_aab_files, install_versioned_maui
+from shared.precommands import PreCommands
+from shared.versionmanager import versions_write_json, get_version_from_dll_powershell
+from test import EXENAME
+
+setup_loggers(True)
+
+precommands = PreCommands()
+install_versioned_maui(precommands)
+
+# Setup the Maui folder
+precommands.new(template='android',
+                output_dir=const.APPDIR,
+                bin_dir=const.BINDIR,
+                exename=EXENAME,
+                working_directory=sys.path[0],
+                no_restore=False)
+
+# Build the APK
+shutil.copy('./MauiNuGet.config', './app/Nuget.config')
+precommands.execute([])
+
+# Remove the aab files as we don't need them, this saves space
+output_dir = const.PUBDIR
+if precommands.output:
+    output_dir = precommands.output
+remove_aab_files(output_dir)
+
+# Copy the xamarinVersion to a file so we have it on the machine
+xamarin_version = get_version_from_dll_powershell(rf".\{const.APPDIR}\obj\Release\{precommands.framework}\android-arm64\linked\Mono.Android.dll")
```

Comment:
nit: might want to call this netandroid_version too to avoid confusion

Reply:
Updated to net_android